packages/bvaughn-architecture-demo/components/console/filters/EventType.tsx

```diff
@@ -3,7 +3,7 @@ import { ConsoleFiltersContext } from "@bvaughn/src/contexts/ConsoleFiltersConte
 import { Event } from "@bvaughn/src/suspense/EventsCache";
 import { useContext } from "react";
 
-import Checkbox from "../../../../../src/ui/components/shared/Forms/Checkbox";
+import { Checkbox } from "../../../../components";
```

Comment:
This import still won't work because it's importing from _outside of the `packages/bvaughn-architecture-demo` directory.

Reply:
Just to be clear: the only package that `packages/bvaughn-architecture-demo` can import from (other than itself) is `packages/shared`.